20200804035648_add_explanation_to_profile_fields.rb

```diff
@@ -0,0 +1,5 @@
+class AddExplanationToProfileFields < ActiveRecord::Migration[6.0]
+  def change
+    add_column :profile_fields, :explanation, :string
```

Comment:
Like I mentioned in #9580, I'll keep adding columns to this table as the need arises.

Reply:
Ah yes, that is a better name.